tutorials/algorithms/02_vqe_advanced_options.ipynb

```diff
@@ -0,0 +1,541 @@
+    "\n",
+    "This section demonstrates how to leverage callbacks in `VQE` to plot the convergence path to the ground state energy with a selected set of optimizers."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we create a qubit operator for VQE. Here we will use the same operator as used in the algorithms introduction, which was originally computed by Qiskit Nature for an H2 molecule."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit.quantum_info import SparsePauliOp\n",
+    "\n",
+    "H2_op = SparsePauliOp.from_list(\n",
+    "    [\n",
+    "        (\"II\", -1.052373245772859),\n",
+    "        (\"IZ\", 0.39793742484318045),\n",
+    "        (\"ZI\", -0.39793742484318045),\n",
+    "        (\"ZZ\", -0.01128010425623538),\n",
+    "        (\"XX\", 0.18093119978423156),\n",
+    "    ]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We then instantiate our `Estimator` of choice for the evaluation of expectation values within `VQE`. In this case, we will use the `qiskit.primitives.Estimator` shipped with the default Qiskit Terra installation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "from qiskit.primitives import Estimator\n",
+    "\n",
+    "estimator = Estimator()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will now see the VQE callback usage over a set of optimizers for comparison. The minimum energy of the H2 Hamiltonian can be found quite easily, so the value of maxiters does not have to be very large. We will once again use `TwoLocal` as our selected trial wavefunction (i.e. ansatz)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Optimization complete      \n"
+     ]
+    }
+   ],
+   "source": [
+    "import numpy as np\n",
+    "from qiskit.algorithms.minimum_eigensolvers import VQE\n",
+    "from qiskit.algorithms.optimizers import COBYLA, L_BFGS_B, SLSQP\n",
+    "from qiskit.circuit.library import TwoLocal\n",
+    "from qiskit.utils import algorithm_globals\n",
+    "\n",
+    "# we will iterate over these different optimizers\n",
+    "optimizers = [COBYLA(maxiter=80), L_BFGS_B(maxiter=60), SLSQP(maxiter=60)]\n",
+    "converge_counts = np.empty([len(optimizers)], dtype=object)\n",
+    "converge_vals = np.empty([len(optimizers)], dtype=object)\n",
+    "\n",
+    "for i, optimizer in enumerate(optimizers):\n",
+    "    print(\"\\rOptimizer: {}        \".format(type(optimizer).__name__), end=\"\")\n",
+    "    algorithm_globals.random_seed = 50\n",
```

Comment:
Do we need to set it in every iteration?

Reply:
Yes, so that the starting point and the conditions for each run are identical when random - we are just looking for what happens with the different optimizers in each loop compared to one another.